site/src/main/scala/org/typelevel/sbt/TypelevelSitePlugin.scala

```diff
@@ -146,6 +150,57 @@ object TypelevelSitePlugin extends AutoPlugin {
       tlSiteApiUrl.value.orElse(javadocioUrl).orElse(fallbackUrl)
     },
     tlSiteHeliumConfig := {
+      // default fontPath and fonts taken from Laika:
+      // instead of allowing for more fonts to be added, all the fonts must be respecified, to avoid "redundant embedding of unused fonts"
+      // as a result, these things have to just be defined again if we want to change them, and we do
+      val laikaFontPath = "laika/helium/fonts"
+      val tlFontPath = "org/typelevel/sbt/site/fonts"
```

Comment:
Oh laika puts it under helium? I didn't think fonts are helium specific. Meh, we can leave it as is.

Reply:
Well, we can't change it anyway right? This is where they are inside the laika jar

Reply:
But yeah, I guess they put it there because the fonts are only used as "part of helium"

Reply:
Right, I meant whether we should put it under helium too, to be consistent.

Reply:
ohhhh

Reply:
Easy change if you wanted it, but don't think it adds much really

Reply:
Exactly. Meh :)